packages/back-end/src/routers/project/project.controller.ts

```diff
@@ -333,7 +325,7 @@ export const putProjectSettings = async (
 
   const { settings } = req.body;
 
-  await updateProjectSettings(id, project.organization, settings);
+  await context.models.projects.updateSettingsById(id, settings);
```

Comment:
Since we have the `project` already in memory, we should not call a `byId` method, since that will require an unnecessary db lookup.